### PR TITLE
adding more API centric content.- general improvements

### PR DESCRIPTION
**Pull request by dougsillars**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [Home](/home)  
 [Getting Started](/getting-started)  
-[Brands](/brands)  
 [Programs](/programs)  
+[Brands](/brands)  
+[Locations](/Locations)
```
Comment (c0derabbit) [inline README.md:5]: let's lowercase `/locations` too so we're consistent

File: brands/README.md [hunk 2]
```diff
@@ -4,24 +4,40 @@ After creating an account and getting access to the dashboard, the next step is
...
   
-The **Brand** object is used to aggregate locations and keep track of the Brand consent. To track real-time transactions at a Brand's locations, you need to obtain consent from authorised personnel (Brand User), and provide the Brand with access to view transactional data. You can do this when creating the Brand (see below).
+The **Brand** object is used to aggregate locations and keep track of the Brand consent. To track real-time transactions at a Brand's locations, you need to obtain consent from authorised personnel (Brand Contact), and provide the Brand with access to view transactional data. You can do this when creating the Brand (see below).
 
 Once you create a **Brand**, it cannot be deleted.
 
 ## Create Brand
 
-To create a new Brand, go to the **Brands** page on the dashboard, click the **+** icon and enter a name. Optionally, you can add a link to the brand logo.
+![Create brand](https://docs.fidel.uk/assets/images/create-brand.png "Create brand")
```
Comment (c0derabbit) [inline brands/README.md:13]: let's use the asset from github

File: brands/README.md [hunk 3]
```diff
@@ -4,24 +4,40 @@ After creating an account and getting access to the dashboard, the next step is
...
+  }'
+```
 
-To automatically request Brand consent, enter the name and email of the Brand User and Fidel will send an email inviting the Brand User to create a Brand account at [clo.fidel.uk](https://clo.fidel.uk). By creating an account the Brand User provides consent and will have access to transactions made by cardholders at participating locations.
+## Brand Consent
 
-If you're funding your own offers, or you've received direct authorisation from a brand, you can now auto-approve the brand's consent – [see here](https://releasenotes.fidel.uk/auto-approve-brand-consent-92004) for more information.
+All brands must be approved by the authorised Brand Contact. To automatically request Brand consent in the dashboard, enter the name and email of the Brand Contact and Fidel will send an email inviting them to create a Brand account at [clo.fidel.uk](https://clo.fidel.uk). To request Brand Consent with the APi, use the [Brand Consent Endpoint](https://reference.fidel.uk/reference#create-brand-user). By creating an account the Brand User provides consent and will have access to transactions made by cardholders at participating locations.
```
Comment (c0derabbit) [inline brands/README.md:30]: typo, `APi` to `API`

Comment (c0derabbit) [reply brands/README.md:30]: `endpoint` should be lowercase

File: cards/README.md [hunk 4]
```diff
@@ -1,30 +1,42 @@
 # Cards
-The Card object holds information about the card details submitted by the user using the web or mobile SDKs. One user can link multiple cards, debit or credit under the same account.
+The Card object holds information about the card details submitted by the user using the web or mobile SDKs. One user may link multiple debit or credit cards using Fidel.
```
Comment (c0derabbit) [inline cards/README.md:2]: don't think we need `using Fidel` here, wdyt?

File: cards/README.md [hunk 5]
```diff
@@ -1,30 +1,42 @@
...
+## Add Card with SDK (preferred)
 
-We never store users' 16-digit card numbers. To identify the user in a transaction object you should use the `cardId` property. After this point only the `cardId` is exchanged between your servers, the card schemes and Fidel API.
+In order to link a card to a program, we recommend that you use our secure and PCI compliant SDKs. These require the user to enter their 16-digit card number and expiry date.  The SDK will be prepopulated with the country code and the `programId` of the Program you want to link their card to. On submission, the card number is tokenised and transmitted directly from our secure pre-built iframe to the API. This way, your servers are never exposed to sensitive information, removing all PCI compliance requirements from your side.
```
Comment (c0derabbit) [inline cards/README.md:6]: we can remove 16-digit, it's not always accurate (amex is just 15)

Comment (c0derabbit) [reply cards/README.md:6]: there's a double space after the 2nd sentence, let's remove that.

File: cards/README.md [hunk 6]
```diff
@@ -1,30 +1,42 @@
...
+
+<div class="info-box">
+  <small>Linking Multiple Cards</small><br/>
+  To facilitate users linking multiple cards, add identication key:value pairs from your system in the metadata field.
+</div> 
+
+We never store users' 16-digit card numbers. To identify the user in a transaction object you should use the `cardId` property. After this point only the `cardId` is exchanged between your servers, the card networks and Fidel's API.
```
Comment (c0derabbit) [inline cards/README.md:13]: we can remove `16-digit` here as well. i think we can just say `We never store card numbers.`

maybe change `you should` -> `you can` so we're friendlier?

File: cards/README.md [hunk 7]
```diff
@@ -1,30 +1,42 @@
...
 
 After the card is linked successfully, we will monitor any purchase made by this card at any of the program’s physical or online locations. The transaction object will be sent to a webhook URL specified by you.
 
-## Add Card
+## Add Card with API
+
+To add a card using the API, you must be PCI Compliant.  Reach out to your Fidel contact for more information.
```
Comment (c0derabbit) [inline cards/README.md:19]: let's remove double space.

Comment (c0derabbit) [reply cards/README.md:19]: should we add some other way of contacting us (support email)?

File: locations/README.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+## Add Locations
+Every Location is linked to a Brand, and the Brand is linked to the Program.
```
Comment (c0derabbit) [inline locations/README.md:2]: kind of, but this is a bit confusing as you add locations to programs (on the dashbard) not brands, and you have to call `POST programs/{programId}/locations` to create a location. so i'd either rewrite or remove this sentence

File: locations/README.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+To add a location, select a Program from the **Programs** page, click the **+** icon and enter the requested info about the location. You can also use the [API to add and modify locations](https://reference.fidel.uk/reference#create-location).
+
+The information required for each location is the Brand (with approved consent), address, postcode, city and country.  If your location has been assigned Merchant IDs (MIDs) from the card networks, you may add them now. Adding MIDs during Location creation will speed the onboarding process.
+
+>Note: If you you use a Payment facilitator, you may be using a [shared MID](https://community.fidel.uk/t/what-is-a-shared-merchant-id-mid/41) that affects how transactions are monitored.  Reach out to Fidel if you use a paymant facilitator and you believe you might have a shared MID.
+
+##### After creating a Program, you can add Locations.
```
Comment (c0derabbit) [inline locations/README.md:10]: you also need a brand (first program, then brand, then location)

File: locations/README.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+![Add locations](https://raw.githubusercontent.com/FidelLimited/docs/master/assets/images/add-locations.png "Add locations")
+
+Once you have added your locations to Fidel, we can begin the process of Location Sync ⚡️. This step requires submitting all of the locations for onboarding with VISA, Mastercard and American Express, so that we may track credit/debit card transactions at each location. During this process, you can monitor and keep track of the status of each location.
```
Comment (c0derabbit) [inline locations/README.md:14]: `VISA` to `Visa`, they don't capitalise

File: programs/README.md [hunk 11]
```diff
@@ -7,41 +7,27 @@ A **Program** is a set of locations that uniquely represent an offline or online
...
-
->Program Sync can take 1-2 weeks, and only one sync per program can be run at a time, so ensure that you are ready to run this process.
-
-![Sync button](https://raw.githubusercontent.com/FidelLimited/docs/master/assets/images/programsync_button.png "Add locations")
-
-Once the sync process is initaited, the Location status is updated to *Syncing*. It then moves to *Active* when we receive confirmation from the schemes that the location has been successfully on-boarded to your program. If the schemes have an issue with a specific location, the status is set to *Failed* and a case is opened to resolve the issue.
-
-For Locations in the live environment, status can be tracked on the dashboard. A green check ✅ means we have received at least one transaction from this Location confirming the active onboarding status. A **RT** icon will show where we can track real-time authorisation transaction and the **RT** will turn green when we receive the first auth transaction from this Location.
+A `programId` will be generated and is used as a data property when using the SDKs to link Cards to this Program. Cards are linked to Programs and consequently to Locations.
```
Comment (c0derabbit) [inline programs/README.md:31]: cards are not linked to locations, but _track_ at participating locations.

File: web-sdk/README.md [hunk 12]
```diff
@@ -53,6 +53,8 @@ fileName:index.html
 ```
 To integrate **Fidel Web SDK** in your website or mobile app, you need to add the script above in your website or mobile web view. For mobile apps, however, we recommend using the [mobile SDKs](/mobile-sdk).
 
+>Hint: Adding user data in the metadata as key:value pairs can reconcile the cardId to a user account in your system.
```
Comment (c0derabbit) [inline web-sdk/README.md:56]: not the cardId, but the metadata id, e.g. if I pass `myUserId: '123'` in metadata, I can match it to user `123` in my system, and [list cards by that id](https://reference.fidel.uk/reference#list-cards-from-metadata-id)

File: cards/README.md (hunk 7 above)
Comment (dougsillars) [reply cards/README.md:19]: added developer@

File: web-sdk/README.md [hunk 13]
```diff
@@ -53,7 +53,7 @@ fileName:index.html
 ```
 To integrate **Fidel Web SDK** in your website or mobile app, you need to add the script above in your website or mobile web view. For mobile apps, however, we recommend using the [mobile SDKs](/mobile-sdk).
 
->Hint: Adding user data in the metadata as key:value pairs can reconcile the cardId to a user account in your system.
+>Hint: Adding user data in the metadata as key:value pairs can simplify reconciliation with your system. For example, adding 'myUserId':'123' will help you match the the added car to your user '123'.
```
Comment (c0derabbit) [inline web-sdk/README.md:56]: typo in `car` -> `card`

**Pull request merged**